sessionctx/variable/session.go

```diff
@@ -997,6 +1039,8 @@ const (
 	TransactionIsolation = "transaction_isolation"
 	TxnIsolationOneShot  = "tx_isolation_one_shot"
 	MaxExecutionTime     = "max_execution_time"
+	IsolationReadLabels  = "isolation_read_labels"
+	IsolationReadEngines = "isolation_read_engines"
```

Comment:
1. For tidb only session variables, it's recommended to add the `tidb_` prefix
2. What does `isolation_read_labels` mean?